### PR TITLE
Add Gradle_Kotlin as option for Build in Download API

### LINKED ISSUE #512: Build tool documentation incomplete
**Describe the bug**
The OpenAPI documentation does not detail all the possible values for the build tool parameter (Gradle Kotlin is missing)

**Expected behavior**
Gradle Kotlin documented

**Actual behavior**
Only Maven and Gradle (Grrovy) are documented

**To Reproduce**
1. See https://code.quarkus.io/q/swagger-ui/#/Download/post_api_download 

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Browser: 
 - OS: 

**Additional context**
(Add any other context about the problem here.)

### PR DESCRIPTION
**Pull request by phillip-kruger**
Fix #512

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

File: src/main/kotlin/io/quarkus/code/rest/OpenAPIFilter.kt [hunk 1]
```diff
@@ -15,7 +15,15 @@ class OpenAPIFilter : OASFilter {
             if (pathItem.get.operationId.equals("extensionsForStream", ignoreCase = true)) {
                 val parameters = pathItem.get.parameters
                 for (parameter in parameters) {
-                    if (parameter.name.equals("stream", ignoreCase = true)) {
+                    if (parameter.name.equals("streamKey", ignoreCase = true)) {
+                        val schema = parameter.schema
+                        schema.enumeration = ArrayList<Any>(validStreamValues)
+                    }
+                }
+            }else if (pathItem.get.operationId.equals("downloadForStream", ignoreCase = true)) {
```
Comment (gsmet) [inline src/main/kotlin/io/quarkus/code/rest/OpenAPIFilter.kt:23]:
```suggestion
            } else if (pathItem.get.operationId.equals("downloadForStream", ignoreCase = true)) {
```
(not sure we check the formatting of Kotlin files though with our current setup)

Comment (gsmet) [reply src/main/kotlin/io/quarkus/code/rest/OpenAPIFilter.kt:23]: Ah, I missed it was not in the core repo, forget about the noise :).

**Review by ia3andy — APPROVED** [review]
LGTM

**Comment by phillip-kruger** [comment]
@ia3andy I am adding some more documentation on the POST as well

**Pull request merged**